core/src/main/java/com/scalar/db/storage/jdbc/JdbcConfig.java

```diff
@@ -167,6 +191,22 @@ public JdbcConfig(DatabaseConfig databaseConfig) {
             ADMIN_CONNECTION_POOL_MAX_TOTAL,
             DEFAULT_ADMIN_CONNECTION_POOL_MAX_TOTAL);
 
+    mysqlVariableKeyColumnSize =
+        getInt(
+            databaseConfig.getProperties(),
+            MYSQL_VARIABLE_KEY_COLUMN_SIZE,
+            DEFAULT_VARIABLE_KEY_COLUMN_SIZE);
+
+    oracleVariableKeyColumnSize =
+        getInt(
+            databaseConfig.getProperties(),
+            ORACLE_VARIABLE_KEY_COLUMN_SIZE,
+            DEFAULT_VARIABLE_KEY_COLUMN_SIZE);
+
+    if (mysqlVariableKeyColumnSize < 64 || oracleVariableKeyColumnSize < 64) {
```

Comment:
I think it's probably better to make `64` a static variable like `MINIMUM_VARIABLE_KEY_COLUMN_SIZE.
We should probably note why 64.

Reply:
@feeblefakie Definitely, thank you! I updated it with the reason based on Mitsu's comment in 1072e98.